include/fe/fe.h

```diff
@@ -596,6 +596,30 @@ class FE : public FEGenericBase<typename FEOutputType<T>::type>
   virtual void init_shape_functions(const std::vector<Point> & qp,
                                     const Elem * e);
 
+  /**
+   * Fills \p dphidxi (and in higher dimensions, eta/zeta) derivative
+   * values for all shape functions, evaluated at points qp in p.  You
```

Comment:
Typo suggests that my fingers were failing rather than my brain, which is kind but inaccurate.

Yeah, they don't have to be quadrature points; I'll change it.